srcpkgs/MoarVM/template

```diff
@@ -1,20 +1,19 @@
 # Template file for 'MoarVM'
 pkgname=MoarVM
-version=2020.11
+version=2021.02
 revision=1
 build_style=configure
 configure_script="perl Configure.pl"
 configure_args="--prefix=/usr --has-libatomic_ops --has-libffi
- --has-libtommath --has-libuv"
+ --has-libtommath"
 hostmakedepends="perl"
-makedepends="libatomic_ops-devel libffi-devel libtommath-devel libuv-devel
- libzstd-devel"
+makedepends="libatomic_ops-devel libffi-devel libtommath-devel libzstd-devel"
```

Comment:
The commit message should mention that you are removing the system libuv for the vendored in one (I believe?), but ideally we could just update our `libuv` and have these packages keep using it...

Reply:
Given the relative importance of libuv to MoarVM, I'd be quite a bit more confident using Raku with the exact version of libuv upstream vendors. As of now, upstream is using [libuv v1.41.0](https://github.com/MoarVM/MoarVM/commit/cf60e095dc1d467e002a2d3cd5c1eaff8219c294).

Raku would probably work fine with a system libuv that differs from the vendored one, and I wouldn't be opposed to keeping the libuv system dep. But nor would I want Raku's usage of libuv to drive updates and patches — IIRC Raku has patched its vendored version of libuv in the past — to Void's libuv.

> ideally we could just update our libuv and have these packages keep using it...

LMK if this is what you want to do, then.

Reply:
Yes, I would prefer that you update `libuv`. As is, if MoarVM can use system libuv with just a configuration argument, we should consider that supported and stick with the system one.